src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -155,12 +153,6 @@ const NavSearchBar = () => {
         </div>
       )}
 
-      {/* @debt use only one UserProfileModal instance with state controlled with redux  */}
-      <UserProfileModal
-        userProfile={selectedUserProfile}
-        show={selectedUserProfile !== undefined}
-        onHide={() => setSelectedUserProfile(undefined)}
-      />
       {/* @debt use only one RoomModal instance with state controlled with redux */}
       <RoomModal
         show={hasSelectedRoom}
```

Comment:
[OOS] But perhaps this can be the next task we take on? As it follows the same pattern as this one.